WcaOnRails/lib/results_validators/generic_validator.rb

```diff
@@ -34,5 +34,17 @@ def reset_state
       @errors = []
       @warnings = []
     end
+
+    protected
+
+    # FIXME: this is kind of a helper that will be needed in several validators as well as in specs.
```

Comment:
I would welcome some feedback on where to put this, as it's used both in the validators and in the specs testing the validators.

Reply:
One way would be defining `personName` on `InboxResult`, so that we can just use `result.personName` everywhere. How does it sound?

Reply:
Sounds like a great idea :)